assignments/tasks.go

```diff
@@ -75,6 +75,11 @@ func handleTasks(ctx context.Context, db database.Database, sc scm.SCM, course *
 		return err
 	}
 
+	// TODO(Espeland): This will create all tasks in the course creator's name.
+	// A possible workaround is to create a new scm client for every repo, and create the issues in one of the students name.
+
+	// TODO(Espeland): Need to think about what happens if an assignment contains two assignments with the same name.
```

Comment:
Do you mean issues instead of assignments?

Reply:
Both this comment, and the previous are solved by QuickFeed being a GitHub app, and therefore both of these todo's should be removed.
I somewhat discuss this in the report.